.github/workflows/mutation-tests.yml

```diff
@@ -13,9 +13,11 @@ jobs:
     strategy:
       matrix:
         dependencies:
-          - "locked"
```

Comment:
Mutation tests should run on locked dependencies: mutation testing framework changes regularly, so we cannot have this instability here